scenarios/README.md

```diff
@@ -401,6 +401,41 @@ The scenario named `custom` can be used to pass any custom filter variable like
 crank --config https://github.com/aspnet/benchmarks/blob/main/scenarios/dotnet.benchmarks.yml?raw=true --scenario custom --profile aspnet-perf-win --variable filter=*LinqBenchmarks*
 ```
 
+## HTTP client benchmarks
+
+These scenarios are running various HTTP client benchmarks.
+
+### Sample
+
+```
+crank --config https://raw.githubusercontent.com/CarnaViire/Benchmarks/http3-benchmarks/scenarios/httpclient.benchmarks.yml --scenario httpclient-kestrel-get --profile aspnet-perf-lin
+```
+*TODO: update to `https://raw.githubusercontent.com/aspnet/Benchmarks/main/scenarios/httpclient.benchmarks.yml`*
+
+### Available scenarios
+
+- `httpclient-kestrel-get`: Execute GET requests to the server. Client is HttpClient, server is Kestrel (minimal API).
+
+#### Arguments
+
+- HTTP Version to use:
+  - `--variable httpVersion="1.1"` (default)
+  - `--variable httpVersion="2.0"`
+  - `--variable httpVersion="3.0"`
+- Number of clients:
+  - `--variable numberOfClients=<N>` (default: `1`)
+- Number of concurrect requests per one client:
+  - `--variable concurrencyPerClient=<N>` (default: `12`)
```

Comment:
This seems like a low default. 

I assume we want to max CPU for this test. As such shouldn't this be much larger?

Reply:
We talked below how we are not building a WRK here. Would we still want to max out the CPU by default?

Reply:
When measuring throughput (as we are here) we generally want to max out CPU on the thing being measured, i.e. httpClient. This serves several purposes: most importantly, it's basically meaningless to compare benchmark throughput results if the CPU Is not maxed. Also, if we can't max the CPU then that usually indicates some sort of contention problem.

Note that when we run ASP.NET server benchmarks (e.g. TechEmpower) we don't care whether the WRK job hits max CPU or not. In fact it's generally better if it doesn't.